packages/polaris-viz/src/components/ComparisonMetric/ComparisonMetric.tsx

```diff
@@ -16,6 +18,18 @@ export function ComparisonMetric({
   accessibilityLabel,
 }: Omit<ComparisonMetricProps, 'dataIndex'>) {
   const selectedTheme = useTheme();
+  const useTrendIndicator = selectedTheme.trendIndicator.useTrendIndicator;
+
+  if (useTrendIndicator) {
```

Comment:
Would you mind checking with UX, we may be able to just use this everywhere and drop the chevron